controllers/uninstall.go

```diff
@@ -65,6 +65,41 @@ var (
 				types.NamespacedName{Name: "assisted-service-sub", Namespace: m.Namespace},
 				schema.GroupVersionKind{Group: "apps.open-cluster-management.io", Kind: "Subscription", Version: "v1"},
 			),
+			// backups.velero.io CRD is removed in 2.5.0
```

Comment:
Why are these in the uninstall list? Shouldn't they be handled in the logic added to the reconciler loop?

Reply:
The reconciler logic does the initial delete. One thing that I was seeing was that without adding them to the uninstall list they were immediately re-created including our installer labels as the reconcile continued

Reply:
this smells like a bug

Reply:
this smells like a bug

Reply:
So, I have added to our blocking logic and they are deleted. However, if they are not also added to our uninstall logic, they are immediately re-created by the reconcile along with our installer labels

Reply:
We need to figure out why that is, otherwise this is reverting my reverting of the deleting of the CRDs in https://github.com/stolostron/multiclusterhub-operator/pull/749/files. The expectation, and what Valentina was expecting, is that the initial delete is the only delete we do.